### PR TITLE
Updates to KitOps homepage

### PR DESCRIPTION
**Pull request by javisperez**
Updates to the texts and layout for the KitOps home page.

### CODE REVIEW

File: docs/.vitepress/theme/components/Home.vue [hunk 1]
```diff
@@ -165,14 +166,70 @@ const subscribeToNewsletter = async () => {
   </video>
 </div>
 
+<div id="whykitops" class="mt-32 md:mt-40 xl:mt-60 px-6 md:px-12 content-container">
+  <h2 class="text-center">Why Kit<span class="font-heading font-extralight">?</span></h2>
+
+  <div class="mt-10 md:mt-14 xl:mt-22 grid grid-cols-1 md:grid-cols-2 gap-4 md:gap-x-4 md:gap-y-[4.5rem] xl:gap-22 max-w-[47.5rem] mx-auto">
+    <div class="h4 !text-gold">Model handoffs are hard.</div>
+    <div class="p2 space-y-4">
+      <p>Moving a model from a Jupyter notebook to an ML tool or development server, then to a production server like Kubernetes is difficult because each tool uses its own packaging mechanism, and requires engineers to repackage the model multiple times. This slows down development and introduces risk.</p>
```
Comment (amisevsk) [inline docs/.vitepress/theme/components/Home.vue:175]: This first sentence reads a little clumsy

File: docs/.vitepress/theme/components/Home.vue [hunk 2]
```diff
@@ -165,14 +166,70 @@ const subscribeToNewsletter = async () => {
...
+      <p>KitOps is an open source MLOps project built to standardize packaging, reproduction, deployment, and tracking of AI / ML models, so it can be run anywhere, just like application code.</p>
```
Comment (amisevsk) [inline docs/.vitepress/theme/components/Home.vue:176]: Maybe `DevOps/MLOps`?

File: docs/.vitepress/theme/components/Home.vue [hunk 3]
```diff
@@ -165,14 +166,70 @@ const subscribeToNewsletter = async () => {
...
+      <div class="mt-8 flex flex-col flex-1 justify-between">
+        <p class="p2">Create a simple manifest file called a Kitfile with your model, dataset and code. Then build and push the ModelKit to a registry for sharing.</p>
+        <a href="/docs/kitfile/kf-overview.html" class="kit-button kit-button-cornflower md:w-fit mt-6">LEARN MORE</a>
+      </div>
+    </div>
+
+    <div class="kit-card flex flex-col">
+      <div class="h4 font-bold !text-gold">3</div>
+      <div class="mt-8 flex flex-col flex-1 justify-between">
+        <p class="p2">Pull the ModelKit into your pipeline, or use <span class="text-gold">kit dev</span> to start working with the model.</p>
```
Comment (amisevsk) [inline docs/.vitepress/theme/components/Home.vue:218]:
```suggestion
        <p class="p2">Pull the ModelKit into your pipeline, or use <span class="text-gold">kit dev</span> to start working with the model locally.</p>
```

File: docs/.vitepress/theme/components/Home.vue [hunk 4]
```diff
@@ -165,14 +166,70 @@ const subscribeToNewsletter = async () => {
...
+
 <div class="mt-32 md:mt-40 xl:mt-60 px-6 md:px-12 content-container">
   <h2 class="text-center">Key feAtuRes</h2>
   <p class="p2 text-center mt-16">Visit our <a href="https://github.com/jozu-ai/kitops" target="_blank" class="underline">GitHub repo</a> for a list of all features and our roadmap.</p>
 
   <ol class="grid grid-cols-1 md:grid-cols-2 gap-x-22 gap-y-16 mt-16 max-w-[960px] mx-auto p1">
     <li>
       <div class="text-off-white">🎁 Standards-based package</div>
-      <p class="p2 mb-4 text-gray-06">A ModelKit package includes models, datasets, configurations, and code. Add as much or as little as your project needs.</p>
+      <p class="p2 mb-4 text-gray-06">A ModelKit package includes models, datasets, configurations, and code in an OCI package. Add as much or as little as your project needs.</p>
```
Comment (amisevsk) [inline docs/.vitepress/theme/components/Home.vue:232]: Nit

```suggestion
      <p class="p2 mb-4 text-gray-06">A ModelKit package includes models, datasets, configurations, and code in an OCI artifact. Add as much or as little as your project needs.</p>
```

File: docs/.vitepress/theme/components/Home.vue [hunk 5]
```diff
@@ -192,7 +249,7 @@ const subscribeToNewsletter = async () => {
     </li>
     <li>
       <div class="text-off-white">🏭 Tags and versions</div>
-      <p class="p2 mb-4 text-gray-06">Each ModelKit is tagged so everyone knows which dataset and model work together.</p>
+      <p class="p2 mb-4 text-gray-06">Each ModelKit is tagged, versioned, and timestamped so everyone knows which dataset and model work together.</p>
```
Comment (amisevsk) [inline docs/.vitepress/theme/components/Home.vue:252]: Small note: on the CLI side, we're not doing timestamping (in order to ensure digests are reproducible)

**Review by amisevsk — APPROVED** [review]
Bunch of optional comments

**Comment by javisperez** [comment]
@amisevsk all these texts were created / editted by @Jwilliamsr i'll ask him to take a look to your comments. Thanks for the replies!

@Jwilliamsr can you please confirm what of these changes should i apply?

File: docs/.vitepress/theme/components/Home.vue (hunk 2 above)
Comment (Jwilliamsr) [reply docs/.vitepress/theme/components/Home.vue:176]: Good catch. This should just say DevOps. @javisperez I updated the G-doc

File: docs/.vitepress/theme/components/Home.vue (hunk 4 above)
Comment (Jwilliamsr) [reply docs/.vitepress/theme/components/Home.vue:232]: Updated g-doc @javisperez

File: docs/.vitepress/theme/components/Home.vue (hunk 5 above)
Comment (Jwilliamsr) [reply docs/.vitepress/theme/components/Home.vue:252]: @javisperez updated g-doc

File: docs/.vitepress/theme/components/Home.vue (hunk 1 above)
Comment (Jwilliamsr) [reply docs/.vitepress/theme/components/Home.vue:175]: Let's keep for now, I'll work on smoothing it out on the next iteration.

**Review by amisevsk — APPROVED** [review]

**Pull request merged**